easybuild/easyconfigs/f/Forge/Forge-21.0.2.eb

```diff
@@ -0,0 +1,44 @@
+##
+# Based on the easyconfig created by the BEAR Software team at the University of Birmingham.
+# Author:    Robert Mijakovic <robert.mijakovic@lxp.lu>
+##
+easyblock = 'Tarball'
+
+name = 'Forge'
```

Comment:
Hmm, perhaps this should be `Arm-Forge`? Not sure...

Reply:
I can rename it. Sounds to me more descriptive.
I got the initial EC from @branfosj and adapted it for a newer release.